src/app/home/home.component.ts

```diff
@@ -29,17 +65,26 @@ export class HomeComponent implements OnInit {
       "data": "Add or Manage Users"
     },
     {
-      "routerlink": "/comming-soon",
+      "routerlink": "/home",
       "icon": "account_balance",
       "title": "Organisations",
       "data": "Add or Manage Organisations"
     },
     {
-      "routerlink": "/comming-soon",
+      "routerlink": "/home",
       "icon": "assessment",
       "title": "Reports",
       "data": "Add or Manage Reports"
     }
   ]
-
+  wholeclick(data){
```

Comment:
What is the use of this function?

Reply:
click for both icon and text in home page